### PR TITLE
Add virtual media and publish raw data

### PR DESCRIPTION
**Pull request by LRichi**
1. Add VirtualMedia for management virtual media
2. Publish rawData for extends custom features any vendors

### CODE REVIEW

File: .idea/gofish.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (stmcginnis) [inline .idea/gofish.iml:1]: And remove these idea files.

File: redfish/virtual_media.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	NotConnectedVirtualMediaConnectedMethod VirtualMediaConnectedMethod = "NotConnected"
+	AppletVirtualMediaConnectedMethod       VirtualMediaConnectedMethod = "Applet"
+	OemVirtualMediaConnectedMethod          VirtualMediaConnectedMethod = "Oem"
+)
+
+// VirtualMedia is used to represent virtual media resource.
+type VirtualMedia struct {
+	common.Entity
+
+	ODataContext        string                      `json:"@odata.context"` // ODataContext is the odata context.
```
Comment (stmcginnis) [inline redfish/virtual_media.go:33]: godoc comments should be above the line they document.

File: redfish/virtual_media.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
+//
```
Comment (stmcginnis) [inline redfish/virtual_media.go:1]: Please rename file to virtualmedia.go to match the object name.

Comment (stmcginnis) [reply redfish/virtual_media.go:1]: If you run tools/gen_schema.sh, that will generate this file under gofiles and will automatically pull in the spec descriptions as docs for each public struct and field.

File: redfish/virtual_media.go [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package redfish
+
+import (
+	"encoding/json"
+	"github.com/LRichi/gofish/common"
```
Comment (stmcginnis) [inline redfish/virtual_media.go:9]: This will need to be updated to the right namespace.

**Comment by LRichi** [comment]
I made changes

**Pull request closed without merging**